s3-bucket.template

```diff
@@ -49,6 +49,33 @@
         "Enabled",
         "Suspended"
       ]
+    },
+    "Ec2Backups": {
```

Comment:
This isn't really about backups, isn't it? It's really about getting a role for the bucket I just created, so I would name this something different, but I suck at naming things, @tinnightcap ?
 - ReturnBucketRole
 - WithRole